.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - '11'
+  - '10'
```

Comment:
@adamwatters Why did we port to node 10? If this is correct we'd need to adjust the nvmrc file too. :) 

I can fix that later... am just curious. Was it because of the dependency issue?